corehq/apps/export/models/new.py

```diff
@@ -636,6 +636,7 @@ class CaseExportInstanceFilters(ExportInstanceFilters):
     sharing_groups = ListProperty(StringProperty)
     show_all_data = BooleanProperty(default=True)
     show_project_data = BooleanProperty()
+    show_deactivated_data = BooleanProperty()
```

Comment:
should this have a default value so that the exports that don't have this set behave properly?

Reply:
I _believe_ it doesn't need a default for the same reason `show_project_data` doesn't need it (and I'm unsure if `show_all_data` needs it), this should point to this function: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/filters/case_list.py#L57